packages/tables/src/styled/StyledHead.ts

```diff
@@ -16,30 +16,40 @@ interface IStyledHeadProps {
 }
 
 /*
- * 1. Prevent <Checkbox> or <OverflowButton> from leaking over the sticky header
- * 2. Replace header row border with a box-shadow that maintains position
+ * 1. Replace header row border with a box-shadow that maintains position
  */
-const stickyStyles = (props: ThemeProps<DefaultTheme>) => {
-  const borderColor = getColorV8('neutralHue', 300, props.theme);
+const colorStyles = ({ theme }: ThemeProps<DefaultTheme>) => {
+  const borderColor = getColor({ variable: 'border.default', theme });
+  const backgroundColor = getColor({ variable: 'background.default', theme });
 
   return css`
-    position: sticky;
-    top: 0;
-    z-index: 1; /* [1] */
-    box-shadow: inset 0 -${props.theme.borderWidths.sm} 0 ${borderColor}; /* [2] */
-    background-color: ${getColorV8('background', 600 /* default shade */, props.theme)};
+    box-shadow: inset 0 -${theme.borderWidths.sm} 0 ${borderColor}; /* [1] */
```

Comment:
Just checking, `theme.shadows.xs(borderColor)` doesn't work here, right?

Reply:
I don't think so. `xs` returns `0 0 0 ${shadowWidths.xs} ${color}`, so even if we add  `inset`, the axis/fill is incorrect.